Readme.md

```diff
@@ -11,6 +11,30 @@
 $ npm install koa-response-time
 ```
 
+## Usage
+
+Basic usage:
+
+    var Koa = require('koa');
+    var responseTime = require('koa-response-time');
+    var app = new Koa();
+
+    app.use(responseTime());
+
+If you need response high resolution in nano time, set `highResolution` option to `true`:
+
+    app.use(responseTime(highResolution: true));
```

Comment:
`highResolution`, maybe just use `hrtime`?

Reply:
make sense, changed to hrtime.